src/util/util.coffee

```diff
@@ -31,4 +31,26 @@ module.exports.makeJsDate = () ->
   # so using regular js here
   `new Date(...arguments)`
 
-module.exports.jsDate = Date
\ No newline at end of file
+module.exports.jsDate = Date
+module.exports.normalizeMillisecondsFieldInString = normalizeMillisecondsFieldInString = (string, matches) ->
+  msString = matches[14]
```

Comment:
It seems a little dangerous that this util function relies on the specific form of the regex that produced `matches`.  If someone changes the regex in one of the places where this is used, and it changes the indexes in the `matches` array, it's easy to miss fixing this util function to match it.

I'd prefer that the calling code just pass in the ms string (`matches[14]`) directly rather than the whole `matches` array.

Reply:
Good point, i'll update that now.